src/app/contextBroker/contextBroker.cpp

```diff
@@ -346,6 +347,11 @@ PaArgument paArgs[] =
 #define ENT_COMPS_V2       2, { "v2", "entities" }
 #define ENT_COMPS_WORD     ""
 
+
+#define IENT                EntityRequest
+#define IENT_COMPS_V2       3, { "v2", "entities", "*" }
+#define IENT_COMPS_WORD     ""
```

Comment:
I - Individual entity , named like https://github.com/telefonicaid/fiware-orion/blob/develop/src/app/contextBroker/contextBroker.cpp#L468 and following ones